.github/workflows/db_export.yaml

```diff
@@ -5,6 +5,8 @@ on:
   push:
     tags:
       - "*"
```

Comment:
Should I remove this trigger?  It has been removed from hapi-pipelines-prod.

Reply:
We need to decide what our release process will be. We will copy to hapi-pipelines-prod what we want to be in prod, but will we still make releases for our own purposes? If we do, then we could consider adding:
  release:
    types: [published]

We would have to name our dev releases in such a way as to avoid creating confusion with hapi-pipelines-prod releases.

Reply:
> We need to decide what our release process will be. We will copy to hapi-pipelines-prod what we want to be in prod, but will we still make releases for our own purposes? If we do, then we could consider adding:
release:
types: [published]

Actually I think we should replace `tags: -"*"` with `types: [published]`, it's what I had [originally](https://github.com/OCHA-DAP/hapi-pipelines/pull/42/files#diff-0ee2f39889321aac41eee6431c0bba5b1b8d22cdb5eac65f7b83de0c97482673R7) but changed it to tags published in a futile attempt to solve the [detached head issue](https://humanitarian.atlassian.net/browse/HAPI-230). In the end Mike fixed it with the idea to use a different branch, so it should be safe to go back to using `types: [published]` - @b-j-mills could you make that change?

@alexandru-m-g does it work for you if we continue our release process as before, or should we make any modifications?

Reply:
For `hapi-pipelines-prod` I've just kept 2 triggers:
- the scheduled one
- the manual one

I don't see any reason for another one on `hapi-pipelines-prod`.

But the changes you make to the triggers and release process for `hapi-pipelines` shouldn't be an issue. I'll just make sure to not bring in the changes related to the triggers.